front-end/src/pages/SearchSettings/SearchSettings.js

```diff
@@ -136,15 +154,22 @@ function SearchSettings() {
       console.log(response.data);
       setOptions([<Option name="Search Location" default="" useStateVariables={response.data}/>,
       <Option name="Distance from Location" type="number" unit="miles" def={5} useStateVariables={response.data}/>,
-      <SliderOption name="Price Range" min={0} max={1000} useStateVariables={response.data}/>,
+      <SliderOption name="Price Range" min={10000} max={1000000} step={100} useStateVariables={response.data}/>,
       <GridOption name="Property Types" options={Object.keys(response.data.PropertyTypes)} useStateVariables={response.data}/>,
       <GridOption name="Amenities" options={Object.keys(response.data.Amenities)} useStateVariables={response.data}/>,
-      <SliderOption name="Number of Rooms" min={1} useStateVariables={response.data}/>,
       <SliderOption name="Number of Beds" useStateVariables={response.data}/>,
       <SliderOption name="Number of Bathrooms" useStateVariables={response.data}/>])
     })
     .catch(function (error) {
       console.log(error);
+      setOptions([<Option name="Search Location" default="" useStateVariables={useStateVariables}/>,
+      <Option name="Distance from Location" type="number" unit="miles" def={5} useStateVariables={useStateVariables}/>,
```

Comment:
hey, u think we should invalidate any negative values? I'm trying to see how to implement that logic, but the only way I can think of is to 
```suggestion
      <Option name="Distance from Location" type="number" unit="miles" def={5} useStateVariables={useStateVariables}/>,
```

Reply:
its all up to you. if you think its uneccesary, then I'll just merge it
```suggestion
      <Option name="Distance from Location" type="number" unit="miles" def={5} useStateVariables={useStateVariables}/>,
```

Reply:
if u think its necessary, this seems to do the trick:
https://stackoverflow.com/a/64933799

I'm gonna try it in a local branch